group/ristretto255.go

```diff
@@ -97,16 +97,24 @@ func (g ristrettoGroup) HashToElementNonUniform(b, dst []byte) Element {
 	return g.HashToElement(b, dst)
 }
 func (g ristrettoGroup) HashToElement(msg, dst []byte) Element {
```

Comment:
To double-check: there are no existing users of Ristretto255 that we're breaking now?

Reply:
With this PR, oprf is the only package in CIRCL that uses ristretto
hard to tell with other external packages.